docs/UserGuide.md

```diff
@@ -173,9 +173,15 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
+**Q1**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
 
+**Q2**: Are the employees listed in the order of id or name?<br>
+**A**: No, the new added employees will be placed from the end of the list.
```

Comment:
Perhaps the phrasing could be more clear? For example: `The employees are not listed in any explicit order. Newly added employees will be placed from the end of the list.`